.github/workflows/ci-medical-portal-ui.yml

```diff
@@ -27,15 +27,17 @@ jobs:
         id: extract_branch
       - uses: actions/setup-node@v4
         with:
-          node-version: '18.17'
+          node-version: '20'
 
-      - run: npm i
-        working-directory: ${{env.working-directory}}${{env.solution-folder}}
+        - run: npm install 
+        working-directory: "shared-portal-ui"
+
+        - run: npm run build 
+        working-directory: "shared-portal-ui"
 
-      - run: npm i
-        working-directory: "shared-portal-ui/projects"
+      - run: npm install && npm ci
```

Comment:
`npm i` and `npm ci` is redundant, just use `npm ci`

https://stackoverflow.com/questions/52499617/what-is-the-difference-between-npm-install-and-npm-ci

Reply:
Fixed this npm install command